data/Bikramjit21.json

```
@@ -24,9 +33,8 @@
       "icon": "SiHashnode"
     },
     {
-      "name": "Subscribed my Youtube Channel",
+      "name": "Subscribe my Youtube Channel",
       "url": "https://www.youtube.com/channel/UCooyvvpFagid_7UHIA-GiRw",
       "icon": "SiYoutube"
     }
```

requires an additional bracket:
```suggestion
    }
    ]
```